veps/sig-compute/10-dra-devices/vep.md

```diff
@@ -0,0 +1,780 @@
+# VEP #10: Support DRA devices in KubeVirt
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+
+## Overview
+
+This proposal is about adding support for DRA (dynamic resource allocation) in KubeVirt.
+DRA allows vendors fine-grained control of devices. The device-plugin model will continue
+to exist in Kubernetes, but DRA will offer vendors more control over the device topology.
+
+## Motivation
+
+DRA adoption is important for KubeVirt so that vendors can expect the same
+control of their devices using Virtual Machines and Containers.
+
+## Goals
+
+- Align on the API changes needed to consume DRA enabled devices in KubeVirt
+- Align on how KubeVirt will consume devices by external DRA drivers
+
+## Non Goals
```

Comment:
Add or reduce the DRA API complexity in KubeVirt

Reply:
Added